runtimevar/runtimevar.go

```diff
@@ -84,7 +84,7 @@ type Snapshot struct {
 // provider-specific, and using them will make that part of your application
 // non-portable, so use with care.
 //
-// See the documentation for the subpackage used to instantiate Variable to see
+// See the documentation for the subpackage used to instantiate Snapshot to see
```

Comment:
I think `Variable` is correct here. They got `Snapshot` from the concrete type, not a sub-package. Maybe "you used" instead of "used"?

Reply:
Done